presto-docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -72,3 +72,12 @@ Lambda Functions
                       CAST(ROW(0.0, 0) AS ROW(sum DOUBLE, count INTEGER)),
                       (s, x) -> CAST(ROW(x + s.sum, s.count + 1) AS ROW(sum DOUBLE, count INTEGER)),
                       s -> IF(s.count = 0, NULL, s.sum / s.count));
+
+.. function:: zip_with(array<T>, array<U>, function<T, U, R>) -> array<R>
```

Comment:
To be consistent with the documentation above, remove the spaces: `function<T,U,R>`